Cookbook/Technical-Documents/SnapshotsTips.md

```diff
@@ -0,0 +1,63 @@
+# Snapshot Testing tips
+
+## What are snapshot tests?
+
+We're testing visual layer of our apps by using snapshots. Snapshots in our case are images of screens and UI components. They are stored in PNG files.
+
+We're using [SnapshotTesting](https://github.com/pointfreeco/swift-snapshot-testing) library from pointfree to record and compare our snapshots.
+
+This kind of tests help us catching any regressions in visual appearance. For example they are very useful when working on a component which might be used on many screens.
+
+## Tips
+
+### Building UI
+
+Until we adopt [SwiftUI previews](https://nshipster.com/swiftui-previews/), snapshots are the fastest way to work iteratively on new UI. 
+Using snapshot tests as feedback mechanism when building UI is much faster than compiling, launching full app, and verifying appearance only then.
+
+Sample workflow for adding a new screen can look like this:
+
+1. **Initial setup**
+      - Add files for a new screen from following templates: BentoBox and RendererSnapshotTests.
```

Comment:
```suggestion
    - Add files for a new screen from the `BentoBox` template. Make sure to tick the "Include Tests" box
```

Reply:
Seems like `BentoRendererSnapshotTests` template is still a thing, so I'll reference both options.

Reply:
It is (it uses the same files), but since you were mentioning a new page using `BentoBox` is the preferred option. The single test template exists only for when you don't add the tests initially but only afterwards